c++/src/kj/async.c++

```diff
@@ -878,6 +886,11 @@ struct FiberStack::StartRoutine {
     ptr |= static_cast<uintptr_t>(static_cast<uint>(arg2)) << (sizeof(ptr) * 4);
     auto stack = reinterpret_cast<FiberStack*>(ptr);
 
+    /// Jump back into the fiber stack constructor
+    if (setjmp(&stack->impl.fiberContext.jumpBuf[0]) == 0) {
```

Comment:
`&jumpBuf[0]` is redundant (kind of like saying `&*jumpBuf`). You should be able to write: `setjmp(stack->impl.originalContext.jumpBuf)`

Similarly for all the other setjmp/longjmp calls below.

Reply:
Ah that makes a lot more sense than what the errors were telling me.